project/ValidatePullRequest.scala

```diff
@@ -199,63 +52,10 @@ object ValidatePullRequest extends AutoPlugin {
     testGrouping in ValidatePR := (testGrouping in Test).value,
     javaOptions in ValidatePR := (javaOptions in Test).value,
 
-    projectBuildMode in ValidatePR := {
-      val log = streams.value.log
-      log.debug(s"Analysing project (for inclusion in PR validation): [${name.value}]")
-      val changedDirs = (changedDirectories in ValidatePR).value
-      val githubCommandEnforcedBuildAll = (githubEnforcedBuildAll in ValidatePR).value
-
-      val thisProjectId = CrossVersion(scalaVersion.value, scalaBinaryVersion.value)(projectID.value)
+    additionalTasks := Seq.empty,
```

Comment:
Should be set in global scope.

Reply:
Ok, I've made the change, but does it really matter? It's only ever read or overridden in the project scope.

Reply:
Allows for it to be overridden in build scope.

It's also just a good habit to form, akin to not leaving a classes fields null after construction.